language/move-vm/types/src/values/mod.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-mod values_impl;
+pub mod values_impl;
```

Comment:
I had to do this in order to use the `Reference` in `lcs.rs`, but hoping that there is a better way?

Reply:
Can we do a `pub use` to re-export the `Reference` so that we don't need to make the entire `value_impl` to be public?

Reply:
good catch @runtian-zhou, 
yeah what's the reason for that? 
@vgao1996

Reply:
Will give that a try, thanks!

I am wondering why the `pub use values_impl::*;` below doesn't already accomplish this...

Reply:
`pub use values_impl::Reference` yields the same error:

```
   Compiling move-vm-types v0.1.0 (/Users/shb/libra/language/move-vm/types)
error[E0603]: module `values_impl` is private
 --> language/move-vm/types/src/native_functions/lcs.rs:9:13
  |
9 |     values::values_impl::Reference,
  |             ^^^^^^^^^^^ this module is private
  |
note: the module `values_impl` is defined here
 --> language/move-vm/types/src/values/mod.rs:4:1
  |
4 | mod values_impl;
  | ^^^^^^^^^^^^^^^^

error: aborting due to previous error
```

Reply:
yeah this is all a bit odd and thinking some more it's probably ok and needed to make value_impl public.
The surface of public functions from there should be pretty low but still it should be public, so no issue probably.
@vgao1996?